src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1318,6 +1319,8 @@ inline void Graph::ExecuteNodeWithCatch(const NodePtr& node, SyncInferRequest* r
 
     try {
         ExecuteNode(node, request, numaId);
+    } catch (const ov::Cancelled&) {
+        throw;
```

Comment:
API side should can receive this exception, why split out one Cancelled exception?

Reply:
`OPENVINO_THROW` will throw `ov::Exception`, `ov_infer_request_wait` captures `ov::Cancelled` and `ov::Exception`. If here `ov::Exception` is throwed, `ov_infer_request_wait` will capture `ov::Exception` and return `ov_status_e::GENERAL_ERROR`, this will cause `ov_infer_request/ov_infer_request_test.cancel` test fails.

Reply:
std::exception <--Inherit-- std::runtime_error <--- ov::Exception <--- ov::Cancelled